config.js

```diff
@@ -261,10 +262,30 @@ exports.updateFromArgv = function() {
         exports.fakerank = parseRank(argv.fakerank);
     }
     exports.bot_command = argv._;
+
+    exports.start_date  = new Date();
```

Comment:
why does this need to be a variable? there is no time between this and the only if where it is used

Reply:
indeed!!
its a reuse of the old code, i didnt notice, thanks! :100:

Reply:
fixed :+1: